src/solvers/solver_utils.hpp

```diff
@@ -247,9 +247,8 @@ TaskStatus SetToZero(const std::shared_ptr<MeshData<Real>> &md) {
         IndexRange kb = cb.GetBoundsK(IndexDomain::interior, te);
         const int nvars = pack.GetUpperBound(b, var()) - pack.GetLowerBound(b, var()) + 1;
         for (int c = 0; c < nvars; ++c) {
-          parthenon::par_for_inner(
-              parthenon::inner_loop_pattern_simdfor_tag, member, kb.s, kb.e, jb.s, jb.e,
-              ib.s, ib.e,
+          parthenon::seq_for(
+              kb.s, kb.e, jb.s, jb.e, ib.s, ib.e,
```

Comment:
Ah, this looks like a potential bug in the original code. I think this should be `PARTHENON_DEFAULT_INNER_LOOP_PATTERN`.

Reply:
Ha, that makes a lot more sense. Maybe its better to fix it in a separate PR

Reply:
I may have done this because the right inner loop pattern didn't always exist previously for the default.